plugins/LadspaEffect/LadspaEffect.cpp

```diff
@@ -241,6 +243,14 @@ bool LadspaEffect::processAudioBuffer( sampleFrame * _buf,
 					for( fpp_t frame = 0; 
 						frame < frames; ++frame )
 					{
+						if (	isnan( pp->buffer[frame] ) ||
```

Comment:
@tresf I need your cmakedefine01 here to go `if (	LMMS_DEBUG ||` or maybe an `LADSPA_DEBUG` that is set by the former. If I try to cmakedefine01 LMMS_DEBUG that isn't working because it's already defined.